contributing/code-style-and-quality-general.md

```diff
@@ -0,0 +1,667 @@
+---
+title:  "Apache Flink Code Style and Quality Guide"
+---
+
+{% include code-style-navbar.md %}
+
+{% toc %}
+
+
+## 1. Tools
+
+We recommend to follow the [IDE Setup Guide](https://ci.apache.org/projects/flink/flink-docs-master/flinkDev/ide_setup.html#checkstyle-for-java) to get IDE tooling configured.
+
+
+### Use inspections in IntelliJ
+
+* Import the inspections settings into the IDE (see IDE setup guide)
+    * TODO: Need to agree on a profile and export it (like checkstyle)
```

Comment:
our docs shouldn't contain TODOS

Reply:
I would throw out this entire section because it isn't remotely well-defined

Reply:
Can we comment this out for now?
I think we should agree upon a set of inspections for all committers/contributors to use. That would help a lot in my opinion.

I'd volunteer with a first suggestion of inspections.